apps/gruenbuch/models.py

```diff
@@ -42,13 +46,16 @@ class GruenbuchOverviewPage(MetadataPageMixin, Page):
 
     page_intro_de = fields.RichTextField(
         blank=True, default="",
-        verbose_name="Grünbuch Overview page introduction")
+        verbose_name="Grünbuch Overview page introduction",
+        help_text=HELPTEXT_RICHTEXT_A11Y)
```

Comment:
You could also use the string directly from apps_blocks, so we only define it once and only have to change it once in case that is needed... so here apps_blocks.HELPTEXT_RICHTEXT_A11Y and then remove it from above.. that should not change the migrations.